galley/pkg/crd/validation/config.go

```diff
@@ -305,17 +305,18 @@ func NewWebhookConfigController(p WebhookParameters) (*WebhookConfigController,
 		createInformerWebhookSource: defaultCreateInformerWebhookSource,
 	}
 
-	galleyNamespace, err := whc.webhookParameters.Clientset.CoreV1().Namespaces().Get(
-		whc.webhookParameters.DeploymentAndServiceNamespace, metav1.GetOptions{})
+	galleyClusterRoleName := "istio-galley-" + whc.webhookParameters.DeploymentAndServiceNamespace
```

Comment:
This is going to cause issues with istiod by coupling it to the istio-galley cluster role name. We need to consider this for future. Will open an issue